Auth0/CredentialsManager.swift

```diff
@@ -183,7 +183,7 @@ public struct CredentialsManager {
                 let newCredentials = Credentials(accessToken: credentials.accessToken,
                                                  tokenType: credentials.tokenType,
                                                  idToken: credentials.idToken,
-                                                 refreshToken: refreshToken,
+                                                 refreshToken: credentials.refreshToken ?? refreshToken,
```

Comment:
would this act as an OR? i.e. if `credentials.refreshToken` is defined, use it. If not, use `refreshToken`

Reply:
Exactly, if `credentials.refreshToken` is not nil will use it.